onnxruntime_extensions/_cuops.py

```diff
@@ -90,6 +90,17 @@ def serialize_attr(cls, attrs):
         return attr_data
 
 
+class StringRemove(CustomOp):
```

Comment:
can you update the doc here: https://github.com/microsoft/onnxruntime-extensions/blob/main/docs/custom_text_ops.md ?

Reply:
Added